### PR TITLE
Added red vignette to the HUD when low on health

### PR DESCRIPTION
**Pull request by Fedic**

### CODE REVIEW

File: Source/Redemption/Public/HealthComponent.h [hunk 1]
```diff
@@ -41,9 +44,18 @@ class REDEMPTION_API UHealthComponent : public UActorComponent
 	UPROPERTY(EditDefaultsOnly)
 	TSubclassOf<UUserWidget> GameOverWidgetClass;
 
+	UPROPERTY(BlueprintAssignable, Category = "Health")
+	FOnHealthLowDelegate OnHealthLow;
+
+	UPROPERTY(BlueprintAssignable, Category = "Health")
+	FOnHealthLowDelegate OnHealthRestored;
```
Comment (mac2648) [inline Source/Redemption/Public/HealthComponent.h:51]: are we using those delegates?

Comment (Fedic) [reply Source/Redemption/Public/HealthComponent.h:51]: No I guess that is a hold-over I forgot to delete.

Comment (mac2648) [reply Source/Redemption/Public/HealthComponent.h]: fair enough

**Review by Entwistle-905 — APPROVED** [review]

File: Source/Redemption/Private/HealthComponent.cpp [hunk 2]
```diff
@@ -63,18 +64,42 @@ void UHealthComponent::Die()
...
+            }
+            else
+            {
+                UE_LOG(LogTemp, Error, TEXT("Failed to create widget"));
+            }
+        }
+       
+    }
+
+
```
Comment (DoopweeSLC) [inline Source/Redemption/Private/HealthComponent.cpp:99]: why the empty lines here between the else if?

Comment (Fedic) [reply Source/Redemption/Private/HealthComponent.cpp:99]: Just a mistake or something I deleted while coding and didn't remove the spaces most likely.

**Pull request merged**